### PR TITLE
[p5.js - hi] Translate 01-05 examples

### PR DESCRIPTION
**Pull request by EshaanAgg**

### CODE REVIEW

**Review by TanviKumar — CHANGES_REQUESTED** [review]

File: src/content/examples/hi/05_Transformation/02_Scale/description.mdx [hunk 1]
```diff
@@ -1,19 +1,9 @@
----
+---
 featuredImage: "../../../images/featured/05_Transformation-02_Scale-thumbnail.png"
-title: Scale
-one-line-description: Modify the coordinate system scale.
+title: स्केल
+one-line-description: समन्वय प्रणाली पैमाने को संशोधित करें
```
[the remark this answers is not in the document]
Comment (EshaanAgg) [reply src/content/examples/hi/05_Transformation/02_Scale/description.mdx]: Made the required changes. Thanks for the review.

**Review by TanviKumar — APPROVED** [review]

**Review by Divyansh013 — APPROVED** [review]

**Comment by TanviKumar** [comment]
@Qianqianye this PR can be merged

**Comment by Qianqianye** [comment]
Thanks, could you resolve the conflicts before we merge? Thanks! @EshaanAgg @TanviKumar @Divyansh013

**Comment by EshaanAgg** [comment]
@Qianqianye @TanviKumar @Divyansh013 Resolved the conflicts!

**Comment by Qianqianye** [comment]
Looks great, thanks!

**Pull request merged**